spec/unit/upgrade_format_task_spec.rb

```diff
@@ -87,22 +85,28 @@ def migrate; end
       end
     end
 
+    it "lists each system description and its status during upgrade" do
+      expected_output = <<-EOF
+Reading "description1" ... successfully upgraded from version 1 to #{SystemDescription::CURRENT_FORMAT_VERSION}.
```

Comment:
I think we are usually using single quotes around names in these kind of messages. Could you check, if that is true, and make sure that the patch is consistent with the rest of the code?